src/features/emails/hooks/usePreviewEmail.ts

```diff
@@ -0,0 +1,9 @@
+import { useNumericRouteParams } from 'core/hooks';
+
+export default function usePreviewEmail() {
+  const { emailId, orgId } = useNumericRouteParams();
+
+  return {
+    previewUrl: `/o/${orgId}/viewmail/${emailId}`,
```

Comment:
Oh does this work? I thought it was necessary to use the `uuid` from the email for this to work, but I guess it works like this if your authenticated (which one always will be in the relevant case)?

Reply:
I was a bit surprised that this worked as well! I didn't really know how I would go about accessing the UUID and so I tried just using the emailId since it seemed to be a serial number.